desktop/src/com/mygdx/game/desktop/DesktopLauncher.java

```diff
@@ -8,5 +8,6 @@
 	public static void main (String[] arg) {
 		LwjglApplicationConfiguration config = new LwjglApplicationConfiguration();
 		new LwjglApplication(new Brainstorming(), config);
+		System.out.println(config.height + " " + config.width);
```

Comment:
Is this supposed to be here?

Reply:
Nope, just a debug line I forgot to remove ...